tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -46,56 +34,46 @@ public void run() {
 
     @Override
     public void process(final Socket connection) {
-        try (final var inputStream = connection.getInputStream();
-             final var outputStream = connection.getOutputStream();
-             final var br = new BufferedReader(new InputStreamReader(inputStream))) {
+        try (final var br = new BufferedReader(new InputStreamReader(connection.getInputStream(), StandardCharsets.UTF_8))) {
+            final var outputStream = connection.getOutputStream();
+
             String requestLine = br.readLine();
-            String resourceName = StaticResourceProcessor.resolveResourcePath(requestLine);
-            String contentType = determineContentType(resourceName);
-            
-            URL resourceUrl = getClass().getClassLoader().getResource(resourceName);
-            if (resourceUrl == null) {
-                String notFoundResponse = buildNotFoundResponse();
-                outputStream.write(notFoundResponse.getBytes());
-                outputStream.flush();
-                return;
+            String[] requestParts = requestLine.split(HEADER_DELIMITER);
+            String httpMethod = requestParts[0];
+            String requestUri = requestParts[1];
+
+            Map<String, String> headers = parseHttpHeaders(br);
+            String body = "POST".equals(httpMethod) ? getBody(headers, br) : "";
+            HttpCookie httpCookie = new HttpCookie(headers.get("Cookie"));
+
+            if (isStaticResource(requestUri)) {
+                StaticResourceProcessor.processStatic(requestUri, outputStream);
+            } else {
+                DynamicRequestProcessor.processDynamic(httpMethod, requestUri, body, httpCookie, outputStream);
```

Comment:
여기서, `StaticResourceProcessor`은 `.processStatic()` 메서드를 가지고. `DyanmicRequestProcessor`은 `.processDynamic()` 메서드를 가지는데요. 어차피 클래스 명에서 Static 혹은 Dynamic Resource Processor임을 표현하고 있으므로, 메서드 명이 단순히 .`process()` 인게 더 괜찮지 않을까요?


```suggestion
                StaticResourceProcessor.process(requestUri, outputStream);
            } else {
                DynamicRequestProcessor.process(httpMethod, requestUri, body, httpCookie, outputStream);
```

이에 대한 모루의 의견이 궁금해요 :)

Reply:
오 인정합니다!! 중복이네요!! 감사해여 ㅎㅎ